src/ln/channelmanager.rs

```diff
@@ -5527,9 +5526,23 @@ mod tests {
 				onion_routing_packet: onion_packet,
 			};
 
-			let err = nodes[1].node.handle_update_add_htlc(&nodes[0].node.get_our_node_id(), &msg).err().unwrap();
-			match err {
-				HandleError{err, .. } => assert_eq!(err, "Remote HTLC add would put them over their reserve value"),
+			if test_recv {
+				let err = nodes[1].node.handle_update_add_htlc(&nodes[0].node.get_our_node_id(), &msg).err().unwrap();
+				match err {
+					HandleError{err, .. } => assert_eq!(err, "Remote HTLC add would put them over their reserve value"),
+				}
+				// If we send a garbage message, the channel should get closed, making the rest of this test case fail.
+				/*assert_eq!(nodes[1].node.list_channels().len(), 1);
```

Comment:
Hmmm, comments of the test isn't voluntary there ?
(uncommented on your branch)

Reply:
Well the test test for the correct behavior but it isnt implemented until the next commit, so the next commit just uncomments it. I can add a comment there for clarity if you want, but its just git history consistency....

Reply:
No, don't care go ahead merge it